### ISSUE TITLE
"MSVC 2015 or later is required." should be ""MSVC 2015 Update 2 or later is required."

### ISSUE THREAD

**Issue by tafia** [briansmith/ring#590]
I have Visual Studio 2015 installed but I couldn't compile ring because it always said it needed MSVC 2015 or later.

I found this check in `limbs.inl`:
```c
#if _MSC_FULL_VER < 190023918
#error "MSVC 2015 or later is required."
#endif
```

I happen to have an older version (installed by my sysadmin):
```
Microsoft (R) C/C++ Optimizing Compiler Version 19.00.23026 for x86
```

I manually removed that check to see how it goes and it finally compiled fine.

So here are my questions/requests:

- what risk do I have to use this older version of the compiler?
- can we make the message more explicit and say that we have version xxx while we are expecting version yyy. It took me a while to figure out that *MSVC 2015* is not enough. Or at least mention which update we need at minimum.

**Comment by briansmith** [comment]
> what risk do I have to use this older version of the compiler?

You copied and pasted the three lines that do the check into your GitHub comment. Did you read the long comment above those three lines? They explain why the check is there. I have no idea which earlier versions of MSVC 2015 work because I personally always upgrade to the latest update right away and I think our testing infrastructure also only provides the latest update. So really, the check is out of date and probably should be requiring Update 3, since that's the only one that has been tested recently.

> can we make the message more explicit and say that we have version xxx while we are expecting version yyy. It took me a while to figure out that MSVC 2015 is not enough. Or at least mention which update we need at minimum.

Yes, that's a good idea. I'll leave this issue open for that.

**Comment by briansmith** [comment]
Thanks for reporting this. I fixed this in 8090623a2e24128a9249eb7ff054ff8b73ea5d62 exactly as reported, which is slightly different than your PR. I'll comment on your PR too.
